department-of-veterans-affairs/caseflow
# Bugfix #4 - Fixed front end rendering of issue edit changes made during Legacy decision workflow

Resolves #{[APPEALS-24068](https://jira.devops.va.gov/browse/APPEALS-24068)}

# Description
Added code to allow refresh of legacy appeal issues so updated issue data reaches the frontend

## Acceptance Criteria

## Testing Plan

1. Sign in as BVAAABSHIRE and open a legacy appeal
2. On Task Actions, click "Ready for Dispatch"
3. On the following screen, select "No Special Issues" and click continue
4. Click on any issue to edit and change MST/PACT checkboxes, dropdown fields, or textboxes
5. Hit save, and when redirected to original page note that saved changes reflect on the frontend

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentationa) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
